docs/source/sawtooth_cli/admin_genesis.rst

```diff
@@ -0,0 +1,52 @@
+Admin Genesis CLI Tool
+======================
+
+Overview
+--------
+
+The genesis CLI tool produces a file for use during initialization of a
+Validator.  A network requires an initial block (known as the genesis block),
```

Comment:
The comma is grammatically correct, but not necessary. I suggest removing the comma.